scss/components/_responsive-embed.scss

```diff
@@ -50,10 +52,14 @@ $responsive-embed-ratio-widescreen: 16 by 9 !default;
 
 @mixin foundation-responsive-embed {
   .responsive-embed, .flex-video {
-    @include responsive-embed;
+    @include responsive-embed( map-get($responsive-embed-ratios,default) );
+
+    $responsive-embed-ratios: map-remove($responsive-embed-ratios,default);
```

Comment:
I've changed this to a new local `$ratios` var. Would it be better to do the following instead of redefining the var or setting a new var? 
```scss
@each $name, $ratio in map-remove($responsive-embed-ratios,default) { ...
```